websites/D/Daily Mail/presence.ts

```diff
@@ -0,0 +1,68 @@
+import { findPage } from "./findPage";
+import { handleNewsPage } from "./newsPage";
+
+const presence = new Presence({
+	clientId: "1217153856665026580",
+});
+
+export const enum Assets {
+	Logo = "https://i.imgur.com/glm7on6.png",
+	exclIco = "https://www.pngmart.com/files/15/Red-Exclamation-Mark-PNG-Transparent-Image-279x279.png",
+	play = "https://static.vecteezy.com/system/resources/previews/000/426/027/original/play-icon-vector-illustration.jpg",
+	pause = "https://icons.iconarchive.com/icons/graphicloads/100-flat/256/pause-icon.png",
```

Comment:
Icons should be 512x512px

Reply:
wait even the small icons should be 512x512?

Reply:
Yeah